tests/testthat/test-sce_to_anndata.R

```diff
@@ -20,7 +23,23 @@ test_that("Conversion of SCE to AnnData works as expected", {
   expect_equal(colnames(colData(sce)), colnames(colData(converted_sce)))
   expect_equal(colnames(rowData(sce)), colnames(rowData(converted_sce)))
 
+  # test that H5 file is created with new assay name
+  # add logcounts
+  logcounts(sce) <- counts(sce)
+  new_anndata_file <- "test_anndata_logcounts.h5"
+  expect_snapshot_file({
+    sce_to_anndata(sce, new_anndata_file, x_assay_name = "logcounts")
+    new_anndata_file
+  })
+
+})
+
+test_that("Conversion of SCE to AnnData fails as expected", {
+
   # check that inputting an improper file name causes a failure
   anndata_bad_file <- "test_anndata"
```

Comment:
Probably should use a tempfile here too, with a bad extension... maybe: 
```suggestion
  anndata_bad_file <- tempfile(fileext = ".rds")
```